domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/GjeldendeVedtaksdata.kt

```diff
@@ -1,25 +1,28 @@
 package no.nav.su.se.bakover.domain.revurdering
 
+import arrow.core.NonEmptyList
 import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.common.between
 import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.grunnlag.Grunnlag
 import no.nav.su.se.bakover.domain.grunnlag.Grunnlagsdata
-import no.nav.su.se.bakover.domain.vedtak.Vedtak
+import no.nav.su.se.bakover.domain.vedtak.VedtakSomKanRevurderes
 import no.nav.su.se.bakover.domain.vedtak.lagTidslinje
 import no.nav.su.se.bakover.domain.vedtak.vilkårsvurderinger
 import no.nav.su.se.bakover.domain.vilkår.Vilkår
 import no.nav.su.se.bakover.domain.vilkår.Vilkårsvurderinger
+import java.time.LocalDate
 import java.util.UUID
 
-data class KopierGjeldendeGrunnlagsdataOgVilkårsvurderinger(
-    private val periode: Periode,
-    private val vedtakListe: List<Vedtak>,
+data class GjeldendeVedtaksdata(
+    val periode: Periode,
+    private val vedtakListe: NonEmptyList<VedtakSomKanRevurderes>,
 ) {
     val grunnlagsdata: Grunnlagsdata
     val vilkårsvurderinger: Vilkårsvurderinger
+    val gjeldendePeriodeTilOriginaltVedtak: Map<Periode, VedtakSomKanRevurderes>
```

Comment:
OriginaltVedtak har vi ikke brukt før? Hva med periodeTilVedtakSomRevurderes? eller kanskje delperiode

Reply:
Hm, det har vi kanskje ikke nei. Ref tidligere kommentar om å ikke vinkle det spesifikt inn mot revurdering.. andre forslag? Kanskje det er dekkende med  bare `gjeldendeVedtak`?

Reply:
Jeg ser de har begynt å bruke eksisterende vedtaksinformasjon i skissene. Men gjeldendeVedtak fungerer vel og like bra.

Reply:
Har refaktorert vekk denne og overlater det til tidslinja å finne ut av hva som er gjeldende for en gitt dato.